integrations/argocd/client.py

```diff
@@ -27,13 +32,22 @@ def __init__(self, token: str, server_url: str):
         self.http_client = http_async_client
         self.http_client.headers.update(self.api_auth_header)
 
+    @staticmethod
+    def get_current_version() -> str:
+        """Function to parse pyproject.toml and retrieve the version of the integration"""
+        logger.info("Getting current integration version from pyproject.toml")
+        with open("pyproject.toml", "r") as toml_file:
+            data = toml.load(toml_file)
+            return data["tool"]["poetry"]["version"]
```

Comment:
we already have this kind of function exposed in the ocean framework. 

see - `version.py` file

Reply:
Ahh I didn't realize there's one already. Removing it then